packages/website-eslint/mocks/path.js

```diff
@@ -206,11 +206,11 @@ export function basename(path, ext) {
   return f;
 }
 
-export function extname(path) {
+function extname(path) {
   return splitPath(path)[3];
 }
 
-export default {
+module.exports = {
```

Comment:
we want to use mocks as cjs instead of modules, to improve build size as almsot all pakcages that we are importing are cjs there is no longer need to convert between formats back and forward

(few have a browser build witch we ar taking advantage of)